index.html

```diff
@@ -42,7 +42,9 @@
   <body>
     <div id="main">
       <header id="home">
-        <img class='logo' src="assets/logo.svg" alt="huddle_icon" />
+        <div class="logo-container">
+          <img class='logo' src="assets/logo.svg" alt="huddle_icon" />

```

Comment:
Atado al comentario en el CSS, esto se podría hacer así:

```suggestion
          <img src="assets/logo.svg" alt="huddle_icon" />
```